views/personal/maritalStatus-edit.pug

```diff
@@ -0,0 +1,47 @@
+extends ../base
+
+block variables
+  -var title = 'Change your Marital Status'
+
+block content
+
+  h1 #{title}
+
+  div
+    p Please choose the correct status from the options listed below.
+
+  form.cra-form(method='post')
+    div
+        fieldset.fieldset
+            legend.fieldset__legend 
+                p Marital Status
+            .radios
+                .radios__item
+                    input.radios__input(id="maritalStatusMarried" name="maritalStatus" type="radio" value="Married" checked=(data && data.personal.maritalStatus.toLowerCase()=='married' || !data) aria-describedby=(errors ? 'maritalStatus_error' : false))
+                    label.radios__label(for="maritalStatusMarried") Married or common-law
+                .radios__item
+                    input.radios__input(id="maritalStatusWidowed" name="maritalStatus" type="radio" value="Widowed" checked=(data && data.personal.maritalStatus.toLowerCase()=='widowed') aria-describedby=(errors ? 'maritalStatus_error' : false))
+                    label.radios__label(for="maritalStatusWidowed") Widowed
+                .radios__item
+                    input.radios__input(id="maritalStatusDivorced" name="maritalStatus" type="radio" value="Divorced" checked=(data && data.personal.maritalStatus.toLowerCase()=='divorced') aria-describedby=(errors ? 'maritalStatus_error' : false))
+                    label.radios__label(for="maritalStatusDivorced") Divorced
+                .radios__item
+                    input.radios__input(id="maritalStatusSeparated" name="maritalStatus" type="radio" value="Separated" checked=(data && data.personal.maritalStatus.toLowerCase()=='separated') aria-describedby=(errors ? 'maritalStatus_error' : false))
+                    label.radios__label(for="maritalStatusSeparated") Separated
+                .radios__item
+                    input.radios__input(id="maritalStatusSingle" name="maritalStatus" type="radio" value="Single" checked=(data && data.personal.maritalStatus.toLowerCase()=='single') aria-describedby=(errors ? 'maritalStatus_error' : false))
+                    label.radios__label(for="maritalStatusSingle") Single
+            if errors
+              span.validation-message #{__(errors.maritalStatus.msg)}
+    div
+      details.
+        <summary><span>Help with editing marital status</span></summary>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+
+    input#redirect(name='redirect', type='hidden', value='personal/maritalStatus')
+
+    .buttons
+      .buttons--left
+        button(type='submit') Change Marital Status
```

Comment:
Not a "right now" thing, but I think we should make the left button bigger when the text is bigger 🤔

Reply:
agreed! it's the flex setting on those buttons right now, but I'll take a look at it!